e107_plugins/gsitemap/admin_config.php

```diff
@@ -44,8 +44,8 @@ class gsitemap_adminArea extends e_admin_dispatcher
 		'main/create'		=> array('caption'=> GSLAN_22, 'perm' => 'P'),
 
 		 'main/div0'        => array('divider'=> true),
-		 'main/import'		=> array('caption'=> 'Import', 'perm' => 'P'),
-		 'main/instructions' => array('caption'=> 'Instructions', 'perm' => 'P'),
+		 'main/import'		=> array('caption'=> GSLAN_23, 'perm' => 'P'),
```

Comment:
Thanks, please use LAN_IMPORT which is in lan_admin.php (global admin lans)

Reply:
Updated